msg/ekf2_timestamps.msg

```diff
@@ -0,0 +1,24 @@
+# this message contains the (relative) timestamps of the sensor inputs used by
+# ekf2. It can be used for reproducible replay.
+
+# timestamp             # ekf2 reference time. This matches the timestamp of the
```

Comment:
Since this field is auto-generated I wanted to find a way to document it. I changed it to hopefully something better.